content/this-month/2023-09/index.md

```diff
@@ -69,6 +69,38 @@ In this section, we give an overview of notable changes to the projects hosted u
 
 In this section, we describe updates to Rust OS projects that are not directly related to the `rust-osdev` organization. Feel free to [create a pull request](https://github.com/rust-osdev/homepage/pulls) with the updates of your OS project for the next post.
 
+### [`mkroening/interrupts`](https://github.com/mkroening/interrupts)
+<span class="maintainers">(Section written by [@mkroening](https://github.com/mkroening))</span>
+
+I created a dependency-free `interrupts` crate, allowing you to temporarily disable interrupts on AArch64, 64-bit RISC-V, and x86-64.
+Two different paradigms allow you to run code with interrupts and synchronize with interrupt handlers running on the same hardware thread (core):
```

Comment:
This sentence is a bit confusing to me. Perhaps you mean "without interrupts"?

Reply:
Yes, thanks for proofreading! Fixed.